content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -47,6 +47,9 @@ Then verify the blob by using `cosign`:
 cosign verify-blob "$BINARY" --signature "$BINARY".sig --certificate "$BINARY".cert
 ```
 
+cosign v1.9.0 is required to be able to use the `--certificate` flag. Please use
```

Comment:
maybe say `For cosign verision => v1.9.0 is required`

Reply:
@saschagrunert , what do you think of @cpanato 's recommendation?

Reply:
@cpanato I'd like to avoid having symbolic pointers in the text, or is there something different you'd suggest?

Reply:
The sentence at its current state works for me